--- a/export_presets.cfg
+++ b/export_presets.cfg
@@ -8,7 +8,7 @@
 custom_features=""
 export_filter="all_resources"
 include_filter=""
-exclude_filter="reports/*,node_modules/*,test/*,addons/gdUnit4/*,tmp/*,export/*,build/*,addons/hornear/*"
+exclude_filter="reports/*,node_modules/*,test/*,addons/gdUnit4/*,tmp/*,export/*,build/*,addons/hornear/*,assets/ui/manada/fondo.png,assets/models/SEPT_JUEGOS_PROTOTIPO_techo1_baseColor.jpg,assets/models/SEPT_JUEGOS_PROTOTIPO_cielorraso_sin_luces.png,assets/models/SEPT_JUEGOS_PROTOTIPO_painted_plaster_wall_diff_1k.jpg,assets/models/SEPT_JUEGOS_PROTOTIPO_piso_baseColor.png,assets/models/SEPT_JUEGOS_PROTOTIPO_textura mopa.png,assets/models/SEPT_JUEGOS_PROTOTIPO_textura balde.png"
 export_path="export/web/index.html"
 patches=PackedStringArray()
 encryption_include_filters=""
